docs/creating-claudie-backup/creating-claudie-backup.md

```diff
@@ -0,0 +1,86 @@
+# Creating Claudie Backup
+
+In this section we'll explain where the state of Claudie is and
+backing up the necessary components and restoring them on a completely
+new cluster.
+
+## Claudie state
+
+Claudie stores its state in 3 different places. 
+
+-   Input Manifests are stored in <b>Mongo</b>.
+-   Terraform state files are stored in <b>MinIO</b>
+-   Locking Mechanism for the state files is implemented via <b>DynamoDB</b>
+
+These are the only services that will have a PVC attached to it, the other are stateless.
+
+## Backing up Claudie
+
+All resources that are deployed or created by Claudie can be identified with the following label:
+
+```
+    app.kubernetes.io/part-of: claudie
+```
+
+!!! note "If you want to include your deployed Input Manifests to be part of the backup you'll have to add the same label to them."
```

Comment:
Should we add this label to all the input manifest examples, so that by default the input manifest is backed-up too?
It's clear, that in the IaC world, this should be implicit, but I wonder if the non-IaC users would expect the backup to include the input manifest.

Reply:
I agree 100%

Perhaps we can open an issue for that

Reply:
I'll create an issue for that

Reply:
https://github.com/berops/claudie/issues/959

Reply:
thanks, guys :)